### PR TITLE
service can describe EC2 intances

### PR DESCRIPTION
**Pull request by amolgautam25**
--- service is able to describe EC2 instances across all regions for the payer level account

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+)
+
+func main() {
+
+	// creating a new cient with us-east-1 region by default
+	client, err := clientpkg.NewClient(accessID, secretKey, "", "us-east-1")
+	if err != nil {
+		fmt.Println("error is %v", err)
+	}
+	assumedRole, err := client.AssumeRole(&sts.AssumeRoleInput{RoleArn: aws.String("arn:aws:iam::173028519319:role/OrganizationAccountAccessRole"), RoleSessionName: aws.String("agautam")})
```
Comment (fahlmant) [inline main.go:27]: Can you remove `173028519319` from the code for now? Make it an empty const like accessID, etc.

Comment (fahlmant) [reply main.go:27]: Session name as well

File: main.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	assumedSecretKey := *assumedRole.Credentials.SecretAccessKey
+	assumedSessionToken := *assumedRole.Credentials.SessionToken
+
+	// for debugging purpose only.
+	//	fmt.Println("new access id : ", assumedAccessKey)
+	//	fmt.Println("new secret key is", assumedSecretKey)
+	//	fmt.Println("new session token\n\n", assumedSessionToken)
+
+	// looping through all the regions
+	regionRange := []string{"us-east-1", "us-east-2", "us-west-1", "us-west-2", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3", "ap-northeast-1", "ap-northeast-2", "ap-south-1", "ap-southeast-1", "ap-southeast-2", "sa-east-1"}
```
Comment (fahlmant) [inline main.go:41]: Please move this list to a const

File: main.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	for _, region := range regionRange {
+		fmt.Println("\n EC2 instances in region ", region)
+		client2, err := clientpkg.NewClient(assumedAccessKey, assumedSecretKey, assumedSessionToken, region)
```
Comment (fahlmant) [inline main.go:44]: Minor nit, client2 to is not very descriptive. Maybe something like `assumedRoleClient`

File: main.go [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+		if err != nil {
+			fmt.Println("error is ", err)
```
Comment (fahlmant) [inline main.go:46]: Need to exit here so that the program doesn't continue

File: main.go (hunk 1 above)
Comment (amolgautam25) [reply main.go:27]: resolved.

File: main.go [hunk 5]
```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	clientpkg "awscleaner/pkg/aws"
```
Comment (fahlmant) [inline main.go:4]: This needs to be the full path to the pkg

`github.com/openshift/aws-account-shredder/pkg/aws`

**Pull request closed without merging**